src/content/docs/infrastructure/host-integrations/installation/install-infrastructure-host-integrations.mdx

```diff
@@ -210,3 +201,7 @@ If you are running services without orchestration or on-premises, choose your sc
 ## Update the agent [#upgrade]
 
 To keep on-host integrations up to date, follow standard procedures to [update the on-host integration package](/docs/infrastructure/host-integrations/installation/update-infrastructure-host-integration-package).
+
+## Technical details [#technical-details]
```

Comment:
@zuluecho9 you've already got a reference to this at the top. Is this repetition necesary?